cpp/zotero_harvester.cc

```diff
@@ -178,9 +188,12 @@ struct HarvesterConfigData {
 };
 
 
-void LoadHarvesterConfig(const std::string &config_path, HarvesterConfigData * const harvester_config) {
+void LoadHarvesterConfig(const std::string &config_path, HarvesterConfigData * const harvester_config,
+                         const IniFile::Section &config_overrides=IniFile::Section())
```

Comment:
Don't see why we need a default parameter here. We're always passing `commandline_args.config_overrides_` to this function.